README.md

```diff
@@ -8,6 +8,7 @@ The name taḥqīq (tah-KEEK) comes from the Arabic for text edition.
 
 - Node.JS v16.x
 - NPM v8.x
+- TinyMCE v5 (can be from CDN)
```

Comment:
please add a note to the 'getting started' section with an example of including tiny from cdn

Reply:
Done, but I put it in "usage" instead since I think that is the point where it is used, rather than "development / getting started".

Also fixed a bug where special characters like < and > were being clobbered when going from read-only to TinyMCE, as Tiny requires them to be escaped as HTML entities.